From 3c6f70e50cb586ac4add4dea627c316242691040 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 13 Jul 2017 09:17:29 -0700
Subject: [PATCH] netload: Include libc net headers unconditionally

This fixes compile with musl, and this conditional
is not needed now a days if we are still using glibc < 1.0
we have other bigger problems

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 sysdeps/linux/netload.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/sysdeps/linux/netload.c b/sysdeps/linux/netload.c
index c275db2..446bddd 100644
--- a/sysdeps/linux/netload.c
+++ b/sysdeps/linux/netload.c
@@ -34,8 +34,6 @@
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 
-#if !defined (_LIBC) && defined (__GNU_LIBRARY__) && __GNU_LIBRARY__ > 1
-/* GNU LibC */
 #include <net/if.h>
 #include <netinet/ip_icmp.h>
 #include <netinet/in.h>
@@ -43,15 +41,6 @@
 #include <netinet/tcp.h>
 #include <netinet/udp.h>
 #include <net/if.h>
-#else /* Libc 5 */
-#include <linux/if.h>
-#include <linux/in.h>
-#include <linux/ip.h>
-#include <linux/icmp.h>
-#include <linux/tcp.h>
-#include <linux/udp.h>
-#endif
-
 
 #ifdef HAVE_IFADDRS_H
 /* needed for IPV6 support */
-- 
2.13.2

